mindmeld/active_learning/classifiers.py

```diff
@@ -0,0 +1,342 @@
+import os
+import logging
+from abc import ABC, abstractmethod
+from collections import defaultdict
+from sklearn.model_selection import StratifiedKFold
+
+from .data_loading import DataBucket
+
+from ..components.nlp import NaturalLanguageProcessor
+from ..constants import ACTIVE_LEARNING_RANDOM_SEED
+
+logger = logging.getLogger(__name__)
+
+
+class Classifier(ABC):
+    def __init__(self, app_path, training_level):
```

Comment:
I think type hints will be useful here, its not clear what type training_level is.

Reply:
Good point, added type hints throughout classifiers.py